packages/waku/src/lib/renderers/dev-worker-impl.ts

```diff
@@ -45,20 +45,27 @@ const configPrivateDir = getEnvironmentData('CONFIG_PRIVATE_DIR') as string;
 
 const resolveClientEntryForDev = (
   id: string,
-  config: { basePath: string },
+  config: { rootDir: string; basePath: string },
   initialModules: ClonableModuleNode[],
 ) => {
   for (const moduleNode of initialModules) {
     if (moduleNode.file === id) {
       return moduleNode.url;
     }
   }
-  const filePath = id.startsWith('file://') ? fileURLToFilePath(id) : id;
-  // HACK this relies on Vite's internal implementation detail.
-  return config.basePath + '@fs' + encodeFilePathToAbsolute(filePath);
+  let filePath = id.startsWith('file://') ? fileURLToFilePath(id) : id;
+  console.log('startFilePath', filePath)
+  if (filePath.startsWith(config.rootDir)) {
+    filePath = filePath.slice(config.rootDir.length, filePath.length);
```

Comment:
Is it just removing a prefix?
```suggestion
    filePath = filePath.slice(config.rootDir.length);
```